### ISSUE TITLE
Official Axon Server nonroot docker image cannot be started

### ISSUE THREAD

**Issue by ViliusS** [AxonIQ/axon-server-se#700]
There is a bug in how official Axon Server nonroot docker image is built. `/axonserver/plugins` folder is not pre-created in advance, so exporting this volume creates it with root permissions which is then not writable by the Axon Server which in turn is run by axonserver user.

```
024-07-24 14:14:46 2024-07-24 11:14:46.980  INFO 1 --- [           main] io.axoniq.axonserver.AxonServer          : Axon Server version 2024.1.0
2024-07-24 14:14:57 ERROR: Unable to create cache directory: ./plugins/cache
2024-07-24 14:14:57 ERROR: Error creating bundle cache.
```

I'm not sure if the docker image is built from 
https://github.com/AxonIQ/running-axon-server/blob/18243f639536939d17b3b30f1854144f5475d6e4/2-docker/0-ee-docker-image/build/Dockerfile but you can see that plugins folder is missing from `mkdir` line.

**Comment by ViliusS** [comment]
Are there any news on this? This has a workaround on Kubernetes, but is a pain to manage on Docker Desktop :(

**Comment by MGathier** [comment]
Thanks for reporting this. We will update the process for building the Docker images and add the "plugins" directory to the image with the correct owner. This will be part of the next release (2024.1.2).

**Comment by ViliusS** [comment]
Confirmed as fixed under 2024.1.2.
